### ISSUE TITLE
Bot should work in ogranization and separate repositories

### ISSUE THREAD

**Issue by rndquu** [ubiquity/ubiquibot#764]
Right now the bot works only if it is installed within an organization.

If the bot is installed in a repository which is not part of an organization then at least the followings features do not work:
- initial labels generation
- payment permit generation

What should be done:
- refactor the bot's code to make sure all of it's features work when the bot is installed in a repository which is not part of an organization

**Comment by pavlovcik** [comment]
- Reference https://github.com/pavlovcik/dao.ubq.fi/issues/21#issuecomment-1717284403

**Comment by Wholesomebruh** [comment]
Also refactor the documentation on setup and working of bot.

**Comment by EtherealGlow** [comment]
@rndquu isnt permit gen for repositories fixed by https://github.com/ubiquity/ubiquibot/issues/719

**Comment by pavlovcik** [comment]
Can't test that if the bot won't price issues.

**Comment by rndquu** [comment]
> @rndquu isnt permit gen for repositories fixed by #719

Yes, chances are that permit generation should work in a personal repository after https://github.com/ubiquity/ubiquibot/issues/719 is implemented.

Anyway current issue also implies:
1. Fixing initial labels generation
2. Check that all other bot's commands work fine in a personal repo
